packages/blade/src/components/Indicator/Indicator.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import type { ReactElement } from 'react';
+import { Highlight, Link } from '@storybook/design-system';
+
+import type { IndicatorProps } from './Indicator';
+import { Indicator as IndicatorComponent } from './Indicator';
+import useMakeFigmaURL from '~src/_helpers/storybook/useMakeFigmaURL';
+import Box from '~components/Box';
+import { Button } from '~components/Button';
+import { getPlatformType } from '~utils';
+
+const Page = (): ReactElement => {
+  const figmaURL = useMakeFigmaURL([
+    {
+      themeTokenName: 'paymentTheme',
+      lightModeURL:
+        'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8224%3A1',
+      darkModeURL:
+        'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8224%3A1',
+    },
+    {
+      themeTokenName: 'bankingTheme',
+      lightModeURL:
+        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=8224%3A0',
+      darkModeURL:
+        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=8224%3A0',
+    },
+  ]);
+
+  return (
+    <>
+      <Title />
+      <Subtitle>
+        Indicators describe the condition of an entity. They can be used to convey semantic meaning,
+        such as statuses and semantical-categories.
+      </Subtitle>
+      <Title>Usage</Title>
+      <Link withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
+        View in Figma
+      </Link>
+      <Highlight language="tsx">{`import { Indicator } from '@razorpay/blade/components' \nimport type { IndicatorProps } from '@razorpay/blade/components'`}</Highlight>
+      <Title>Example</Title>
+      <Subtitle>You can change the properties using the controls in the table below.</Subtitle>
+      <Primary />
+      <Title>Properties</Title>
+      <ArgsTable story={PRIMARY_STORY} />
+      <Stories />
+    </>
+  );
+};
+
+const meta: Meta<IndicatorProps> = {
+  title: 'Components/Indicator',
+  component: IndicatorComponent,
+  args: {
+    accessibilityLabel: 'Status OK',
+    children: 'Success',
+    intent: 'neutral',
+    size: 'medium',
+  },
+  argTypes: {},
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+};
+
+const IndicatorTemplate: ComponentStory<typeof IndicatorComponent> = ({ ...args }) => {
+  return <IndicatorComponent {...args} />;
+};
+
+export const Default = IndicatorTemplate.bind({});
+
+export const WithoutLabel: ComponentStory<typeof IndicatorComponent> = ({ ...args }) => {
+  return <IndicatorComponent {...args} />;
+};
+WithoutLabel.args = {
+  children: undefined,
```

Comment:
children by default will be undefined right? do we need to explicitly pass it as undefined?

Reply:
It should be but I'd specified the default value of `children` above in args 😅